build.gradle

```diff
@@ -5,6 +5,7 @@ buildscript {
         mavenCentral()
         gradlePluginPortal()
         maven { url 'https://repo.spring.io/plugins-release-local' }
+        maven { url "https://repo.spring.io/milestone" }
```

Comment:
I wonder why do we need this since I don't see you have added any `M` dependency. 
You have just removed two deps and added one with `backendVersion = '0.0.3'`.
So, why do we need `milestone` repo then?

Reply:
We don't. I got carried away with copying code. I'll remove it.